testimonials/serializers.py

```diff
@@ -11,7 +11,11 @@ class AttestationSerializer(serializers.ModelSerializer):
     avatar = serializers.URLField(source="avatar.url")
     avatar_small = serializers.URLField(source="avatar_small.url")
     avatar_medium = serializers.URLField(source="avatar_medium.url")
-    cover = serializers.URLField(source="cover.url")
```

Comment:
if you set it to the [following](https://www.django-rest-framework.org/api-guide/fields/#allow_null) it should have the same effect 
```
cover = serializers.URLField(source="cover.url",
        allow_null=True,
        read_only=True,
        help_text="Get the cover image URL")
```

Reply:
It does not; I still get a 500 error with the change made. This is trying to reference an attribute in the resulting FieldFile object so a null here means it's trying to pull from an incomplete object.